docs/lab/workflow/order-unregistered-testkit.mdx

```diff
@@ -0,0 +1,92 @@
+---
+title: "Ordering an Unregistered Testkit"
+---
+
+Vital provides a way to order testkits for other patients (registrable testkits). 
+
+Under the [normal order flow](/api-reference/lab-testing/create-order), you place an order for a specific patient. This order is then always associated with that patient and cannot be passed on to someone else.
+
+In order to allow for `testkits` to be purchased and passed on to another patient, Vital provides the `registrable testkit` flow.
+
+<Warning>
+  Note that this only applies to `testkits`, not `walk-in phlebotomy` or `at-home phlebotomy`.
```

Comment:
These are references to the product feature, not to a code entity.

Reply:
Not sure what your point is here

Reply:
AH I see